lib/sisimai/bite/email/einsundeins.rb

```diff
@@ -0,0 +1,143 @@
+module Sisimai::Bite::Email
+  module EinsUndEins
+    # Sisimai::Bite::Email::EinsUndEins parses a bounce email which created by
+    # 1&1. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/EinsUndEins.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->